### ISSUE TITLE
Once game starts, connection problems

### ISSUE THREAD

**Issue by Verazon** [Regalis11/Barotrauma#8393]
The server is being run with a dedicated host using the steacmd Barotrauma Server

This is the output in the console from when a player connects.
`Failed to write a network message for the client "John Halo"! {Object reference not set to an instance of an object.}
   at Barotrauma.Networking.GameServer.ClientWriteLobby(Client c) in <DEV>\Barotrauma\BarotraumaServer\ServerSource\Networking\GameServer.cs:line 1833
   at Barotrauma.Networking.GameServer.ClientWrite(Client c) in <DEV>\Barotrauma\BarotraumaServer\ServerSource\Networking\GameServer.cs:line 0
   at Barotrauma.Networking.GameServer.Update(Single deltaTime) in <DEV>\Barotrauma\BarotraumaServer\ServerSource\Networking\GameServer.cs:line 605`

TL:DR;

Players are unable to join an on-going game, it most likely black screens them and boots them out whilst I see this error on the dedicated server console.

**Comment by Verazon** [comment]
[img]https://i.imgur.com/3KygSeZ.png[/img]

This is what connected players see.

**Comment by Regalis11** [comment]
Duplicate of https://github.com/Regalis11/Barotrauma/issues/8381
